main/src/com/google/refine/expr/functions/strings/Reinterpret.java

```diff
@@ -89,20 +89,20 @@ private Object reinterpret(String str, String decoder, String encoder) {
                 result = new String(bytes, encoder);
             }
         } catch (UnsupportedEncodingException e) {
-            return new EvalError(ControlFunctionRegistry.getFunctionName(this) + ": encoding '" + encoder + "' is not available or recognized.");
+            return new EvalError(ControlFunctionRegistry.getFunctionName(this) + ": target encoding '" + encoder + "' is not available or recognized.");
         }
                         
         return result;
     }
     
     @Override
     public String getDescription() {
-        return "Returns s reinterpreted thru the given encoder.";
+        return "Returns s reinterpreted thru the given target encoder and optional source decoder.";
     }
     
     @Override
     public String getParams() {
-        return "string s, string encoder";
+        return "string s, string target encoder, string source decoder";
```

Comment:
This might read better for our users as the enhancement was worded, instead of my PR wording?
```suggestion
        return "string s, string target encoding, string source encoding";
```